.gitmodules

```diff
@@ -7,3 +7,6 @@
 [submodule "cpython"]
 	path = cpython
 	url = https://github.com/python/cpython.git
+[submodule "pybind11"]
```

Comment:
Why does pybind11 need to be added as a submodule?

Reply:
I thought it needed to be, but pip installing it via requirements.txt is actually sufficient.